content/docs/user-guide/interactivity/physics/debugging.md

```diff
@@ -87,3 +88,15 @@ You must enable the [ImGui Gem](/docs/user-guide/gems/reference/debug/imgui) to
 ## Debug Options in the PhysX Configuration 
 
 You can also specify debug settings in the **PhysX Configuration** tool. See [Debugger Configuration](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-debugger/).
+
+## Enable additional checks and error reporting in PhysX SDK
+
+You can make the profile configuration of O3DE use the **checked** version of PhysX SDK library. PhysX will perform additional checks to detect invalid parameters, API race conditions and other incorrect uses of the API which might otherwise cause mysterious crashes or failures in simulation. The benefit is to enable the same safety checks as in debug configuration but without having to run O3DE on debug, where low framerates could spoil the simulation.
```

Comment:
uber nit: I think we should say: "_...version of **the** PhysX SDK library_"

Reply:
_The benefit is to enable the same safety checks as in debug configuration but without having to run O3DE on debug, where low framerates could spoil the simulation._ 

suggestion:

_The benefit of doing this is the same safety checks from the debug configuration are enabled without having to run O3DE in debug, where low framerates could impact the simulation._

Reply:
Thanks, changed.